apps/docs/licensing.md

```
@@ -0,0 +1,18 @@
+- Use Task Master for any purpose (personal, commercial, academic)
+- Modify the code
+- Distribute copies
+- Create and sell products built using Task Master
+
+## ❌ Not Allowed:
+
+- Sell Task Master itself
+- Offer Task Master as a hosted service
+- Create competing products based on Task Master
+
+{/* See the [LICENSE](../LICENSE) file for the complete license text. */}
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)